pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go

```diff
@@ -61,14 +61,19 @@ func (pl *NodeUnschedulable) Name() string {
 func (pl *NodeUnschedulable) Filter(ctx context.Context, _ *framework.CycleState, pod *v1.Pod, nodeInfo *framework.NodeInfo) *framework.Status {
 	node := nodeInfo.Node()
 
+	if !node.Spec.Unschedulable {
+		return nil
```

Comment:
return nil or return a status?

Reply:
return nil is ok, because the nil is also considered as success，consistent with other plugins，

```
// Code returns code of the Status.
func (s *Status) Code() Code {
	if s == nil {
		return Success
	}
	return s.code
}

```

Reply:
if !node.Spec.Unschedulable，that means  node is schedulable, so we can just return

Reply:
you are right.

/lgtm